recipe/meta.yaml

```diff
@@ -1,30 +1,32 @@
 {% set name = "apispec" %}
-{% set version = "4.7.1" %}
-{% set sha256 = "79029486d36a0d7f3c659dbf6ae50a91fbed0c22dcd5376f592e076c130bc7f9" %}
+{% set version = "6.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: c0846f8eaa5119c46b2ecfe9bc24ed19dba8845f8655d00b51ddd296a10ea4cb
+
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - wheel
     - setuptools
```

Comment:
```suggestion
```
`wheel` and `setuptools` are not needed when a backend other than setuptools is used.
The backend is specified in https://github.com/marshmallow-code/apispec/blob/6.6.0/pyproject.toml#L57-L59.